src/l500/ac-trigger.cpp

```diff
@@ -966,6 +977,45 @@ namespace ivcam2 {
 
                         ivcam2::read_fw_table( *hwm, cal_info.table_id, &cal_info );
 
+                        try
```

Comment:
This should be in `depth_to_rgb_intrinsics`

Reply:
We read all the rest of the tables here (0x313 and registers) so why this table need to be read on depth_to_rgb_calibration?

Reply:
For symmetry -- if `depth_to_rgb_calibration` calculates Krgb from Kth, it should also be able to determine Kth from Krgb. It already does some other reading of tables.

Reply:
done